source/basic.tex

```diff
@@ -5404,6 +5404,8 @@
 
 \rSec1[basic.exec]{Program execution}
 
+\indextext{program execution|(}%
+
 \rSec2[intro.execution]{Sequential execution}
```

Comment:
If you're going to have whitespace here anyway, then no need to bother escaping the newline. Alternatively, remove the blank line.

Reply:
fixed